packages/open-next/src/core/util.ts

```diff
@@ -28,7 +28,11 @@ import {
 overrideNextjsRequireHooks(NextConfig);
```

Comment:
@vicb These 2 uses `require.resolve` and are present on next >=13.4.13 and next <=13.5.1